DQMOffline/EGamma/plugins/PhotonDataCertification.cc

```diff
@@ -85,17 +85,18 @@ float PhotonDataCertification::invMassZtest(string path, TString name,
   if (TestElem == 0) return 0;
   TH1F *TestHist = TestElem->getTH1F();
   if (TestHist == 0) return 0;
+  RooMsgService::instance().setGlobalKillBelow(RooFit::WARNING) ;
```

Comment:
It looks like Roo is very thread-hostile. There isn't even the standard ROOT locks anywhere.